_posts/2020-08-24-changelog-129.markdown

```diff
@@ -0,0 +1,91 @@
+    * Improve package name validation during the new project creation
+
+* Lints & inspections improvements:
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5889 -->
+    * Move all `rustc`-related lints to the newly created separate "Lint" group in inspection settings
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5891 -->
+    * Support the `nonstandard_style` attribute for naming inspections
+    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5890 -->
+    * Now the `While true loop` inspection takes into account the `while_true` lint [level](https://doc.rust-lang.org/rustc/lints/levels.html#lint-levels)
+      and has the suppression quick fix to add `#[allow(while_true)]` attribute
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5865 -->
+    * Check if the `derive` attribute is placed on a supported item (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5940 -->
+* Suggest the run configuration name from its command
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5645 -->
+* Support the `Complete Current Statement` action for function and structs (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5828 -->
+* Allow folding of consecutive `impl Iterator<Item=X>`s in a method chain hints (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5760 -->
+* Show error message when trying to run with coverage on non-nightly channel
+  (instead of forcing to run it on the nightly channel)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5885 -->
+* Update icons for Cargo, REPL and `cargo-generate`
+
+## Performance Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5926 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5912 -->
+* Slightly optimize name resolution in some cases
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5877 -->
+* Don't insert double parentheses after completion
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5870 -->
+* Don't provide the `Change return type` quick fix in a lambda with an implicit return type (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5937 -->
+* Now the `Attach file to a module` quick fix takes into account existing `mod` items, comments and attributes when inserting the new `mod` item (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5873 -->
+* Improve autocompletion of the `cargo` command in the [Run Anything](https://www.jetbrains.com/help/clion/rust-support.html#cargo-support) popup
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5641 -->
+* Don't annotate user-defined attribute macros that override built-in ones (by [@Stzx])
+
+## Internal Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5932 -->
+* Initial regression testing
```

Comment:
Let's add something like `It should help to catch false-positive errors in new changes automatically`